src/components/ui/form/Select.tsx

```diff
@@ -0,0 +1,105 @@
+import { Controller, RegisterOptions, useFormContext, UseFormReturn } from 'react-hook-form'
```

Comment:
Currently we only have `Input`, `Checkbox` and a few others. `Select` would complete the lineup. But the standard `<select><option></option></select>` elements look very out of place amidst the rest of our elements, so I co-opted our `DropdownMenu` component and made it compatible with react-hook-form using `Controller`.

Reply:
we're using DaisyUI for styling: https://daisyui.com/components/select/

Reply:
I updated the classNames to use "select" classes instead of "btn" ones. Hope that was what your suggestion meant!